### ISSUE TITLE
Is the dataset ADE20K means ADEChallengeData2016 or not ?

### ISSUE THREAD

**Issue by xhding1997** [HRNet/HRNet-Semantic-Segmentation#218]
I have download ADEChallengeData2016 and unzip it.
The structure is:

---ADEChallengeData2016
   --annotations
        -validation
        -training
   --images
        -validation
        -training

It doesn't look the same as the README in this repository.
├── ade20k
│   ├── train
│   │   ├── image
│   │   └── label
│   └── val
│       ├── image
│       └── label

**Comment by hadipash** [comment]
Yes, it is ADEChallengeData2016 dataset, and it needs to be converted first:
https://github.com/openseg-group/openseg.pytorch/tree/master/lib/datasets/preprocess/ade20k
